DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+
+![The "New personal access token" page on GitHub, with those four scopes checked.][i:New personal access token]
+
+Then, you can click "Generate token" at the bottom of the page. You'll get a token that looks like `745b0dcc742dedf87c67ff075f65eb87295e205f`. Keep that around.
+
+### Giving that personal access token to `npm`
+
+Next, open a command prompt (Terminal.app on the Mac; PowerShell on Windows) and run
+
+```sh
+npm login --registry=https://npm.pkg.github.com
+```
+
+ * When it asks for a username, enter your GitHub username.
+ * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
+ * When it asks for your email, enter your GitHub email. 
+
+![Powershell after running "npm login --registry=https://npm.pkg.github.com". The output "Logged in as helloworld12321demo on https://npm.pkg.github.com/" is shown.][i:Logging in to npm]
+
+Some sticking-points:
+
+1. For security reasons, what you type into the "password" field is invisible.
+2. To paste text into PowerShell, right-click once.
+3. On github.com/settings/tokens, there personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
+
+![github.com/settings/tokens with the extra space character highlighted][i:The gosh-darn whitespace]
+
+If you run into trouble, you can always fall back to typing the personal access code by hand.
+
+## Installing all of the third-party libraries
+
+Buzz About uses code from several third-party packages. You can install all of these packages at once using `npm`.
+
+Open up the command prompt. First, you'll want to navigate to the folder where you downloaded Buzz About. You can do this by running
+
+```PowerShell
+Set-Location C:\wherever\you\downloaded\buzz-about
+```
+
+on Windows, or
+
+```sh
+cd /wherever/you/downloaded/buzz-about
+```
+
+on a Mac.
+
+Then, run
+
+```sh
+npm install
+```
+
+to download all of the third-party libraries.
+
+![PowerShell after running "Set-Location D:\joe\Documents\buzz-about" and "npm install". The command is halfway through running.][i:Running npm install]
+
+Note that the third-party libraries are installed within this project&mdash; they're downloaded into a folder named `buzz-about/node_modules`. If you have another project on your computer, it will have its own set of libraries.
+
+<!-- Links: -->
+[Paul Friederichsen]: https://github.com/floogulinc
+[Prof. Nic McPhee]: https://github.com/nicmcphee
+[Prof. KK Lamberty]: https://github.com/kklamberty
+[GitHub]: https://github.com
+[Minnesota Pollinators]: https://github.com/mn-pollinators
+[Student developer pack]: https://education.github.com/pack
+[Sign up for GitHub]: https://github.com/join
+[Visual Studio Code]: https://code.visualstudio.com/
+[Live share]: https://docs.microsoft.com/en-us/visualstudio/liveshare/
+[Install git for Windows]: https://git-scm.com/download
+[GitKraken]: https://www.gitkraken.com/
```

Comment:
```suggestion
[GitKraken]: https://www.gitkraken.com/git-client
```

We can link to the git client specifically

Reply:
Oh, as opposed to the boards and stuff? Nice!